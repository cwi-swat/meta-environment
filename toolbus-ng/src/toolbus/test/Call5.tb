process A(S : str) is printf("%s A\n", S)

process B(S : str) is printf("%s B\n", S)

process Main1 is 
let B : str
in
    B := "B" .
    B("Main1") .
    if false then B("Main1: true case") else B("Main1: false case") fi
endlet

process Main2(B : str) is 
    B("via Main2") 

toolbus(Main1, Main2("A"))

