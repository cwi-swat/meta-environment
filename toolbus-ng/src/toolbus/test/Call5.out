"via Main2" A
"Main1" B
"Main1: false case" B
